### PR TITLE
Make http async timeout for nb configurable

### PR DESCRIPTION
**Pull request by timofei-durakov**
- New property `web.request.asyncTimeout` was introduced
- Added to confd templates.

### CODE REVIEW

File: confd/templates/northbound/northbound.properties.tmpl [hunk 1]
```diff
@@ -15,6 +15,8 @@ kafka.hosts={{ getv "/kilda_kafka_hosts" }}
 
 server.contextPath=/api
 
+web.request.asyncTimeout=30000
```
Comment (surabujin) [inline confd/templates/northbound/northbound.properties.tmpl]: Other NB options are starts with `northbound.` why this one differ?

File: services/src/northbound-service/northbound/src/test/resources/northbound.properties [hunk 2]
```diff
@@ -12,6 +12,8 @@ kafka.hosts=kafka.pendev:9092
 
 server.contextPath=/api
 
+web.request.asyncTimeout=30000
```
Comment (surabujin) [inline services/src/northbound-service/northbound/src/test/resources/northbound.properties]: Can we have "default" value for this option? So its presence in property file will become optional.

File: confd/templates/northbound/northbound.properties.tmpl (hunk 1 above)
Comment (timofei-durakov) [reply confd/templates/northbound/northbound.properties.tmpl]: even from this file I could see a plenty of options that starts with something other than northbound prefix

File: services/src/northbound-service/northbound/src/test/resources/northbound.properties (hunk 2 above)
Comment (timofei-durakov) [reply services/src/northbound-service/northbound/src/test/resources/northbound.properties]: why not to follow pattern that is used here?

File: confd/templates/northbound/northbound.properties.tmpl (hunk 1 above)
Comment (surabujin) [reply confd/templates/northbound/northbound.properties.tmpl]: ok

**Pull request merged**